modules/openapi-generator/src/main/resources/php-nextgen/api.mustache

```diff
@@ -164,7 +164,9 @@ use {{invokerPackage}}\ObjectSerializer;
      *
      * @throws ApiException on non-2xx response or if the response body is not in the expected format
      * @throws InvalidArgumentException
+{{#returnType}}
      * @return {{{vendorExtensions.x-php-doc-return-type}}}
+{{/returnType}}
```

Comment:
in other languages such as java, the doc is also completely omitted if it's void so make senses php follows the same.